tox.ini

```diff
@@ -148,7 +148,7 @@ deps =
     celery40: celery>=4.0,<4.1
     celery41: celery>=4.1,<4.2
     celery42: celery>=4.2,<4.3
-    ddtracerun: redis
+    ddtracerun: redis==2.10.6
```

Comment:
can probably just do `redis>=2.6,<3.0` here, we do want to make sure we are compatible with `2.10.7` if it comes out, or `2.11.0`

Reply:
I'd say that the version of redis shouldn't matter here. We use redis as a tool to test `ddtrace-run` and it doesn't make sense for the test to be dependent on any version of redis (so long as it works).

Reply:
sure, makes sense